cmd/webhook/main.go

```diff
@@ -134,6 +141,15 @@ func NewDefaultingAdmissionController(ctx context.Context, cmw configmap.Watcher
 }
 
 func NewValidationAdmissionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+	// Decorate contexts with the current state of the config.
+	store := defaultconfig.NewStore(logging.FromContext(ctx).Named("config-store"))
+	store.WatchConfigs(cmw)
+
+	// Decorate contexts with the current state of the config.
+	ctxFunc := func(ctx context.Context) context.Context {
+		return store.ToContext(ctx)
+	}
```

Comment:
If you're APIs defaulting is now dependent on the store/config in the context - you'll need to update the conversion webhook context decorator - it will invoke defaulting on the way out

Reply:
that is good to know!

Reply:
done: https://github.com/knative/eventing/pull/2651/files#diff-58de452513b7e8d8d3cfea23eb4ae6a8R340